modules/joda/src/test/scala/pureconfig/module/joda/configurable/ConfigurableSuite.scala

```diff
@@ -115,4 +156,30 @@ object ConfigurableSuite {
         localDateTime <- localDateTimeArbitrary.arbitrary
         zoneId <- zoneIdArbitrary.arbitrary
       } yield localDateTime.toDateTime(zoneId))
+
+  implicit val instantArbitrary: Arbitrary[Instant] =
+    Arbitrary(Arbitrary.arbitrary[Long].map(new Instant(_)))
+
+  implicit val intervalArbitrary: Arbitrary[Interval] =
+    Arbitrary(Arbitrary.arbitrary[(DateTime, DateTime)]
+      .filter(i => i._1.isBefore(i._2))
```

Comment:
Suggestion: Comparing the intervals and put the lesser in _1 and greater in _2. That will generate valid cases twice as fast.